README.md

```diff
@@ -1 +1,5 @@
-# Hello-Microverse
```

Comment:
- [ ] Add a professional readme file and use the template that the previous reviewer mentioned
- [ ] You have conflicts on your branch kindly try to fix them
- [ ] Set up linters correctly, use the example provided by the previous reviewer
- [ ] Add a meaningful pull request which has a descriptive title and a sumarry